src/components/BookList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Book from './Book'
+import CreateNewBook from './CreateNewBook ';
+
+const BookList = () => (
+  <div className="container max-width-md">
+    <Book title="Hoom Hoom" author="Blah 2"></Book>
+    <Book title="Zing Zing" author="Blah 2"></Book>
+    <Book title="Bazingazing" author="Blah 2"></Book>
```

Comment:
**Changes Required**
Good effort implementing the requested changes by the previous Code Reviewer. However;
- [ ] The call to the Book component should be self closing. For instance, `<Book />`. This is one of React's good practice. Let us please try and change to this.
- [ ] You can create a dummy array to hold the title and author, then use the Array.prototype.map method to iterate it and pass the title and author as props to the Book Component.